.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Go
+
+on: [push, pull_request]
+
+jobs:
+  shellcheck:
+    runs-on: [ubuntu-latest]
+    steps:
+
+      - uses: actions/checkout@v2
+      - name: Run ShellCheck
+        run: |
+          shellcheck scripts/*.sh
+
+      - name: Run gofmt
+        run: |
+          ./scripts/checks.sh
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x, 1.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          go get gotest.tools/gotestsum
+
+      - name: modVerify
+        run: go mod verify
+
+      - name: modTidy
+        run: go mod tidy
+
+      - name: vet
+        run: go vet ./...
+
+      - name: gotestsum
+        run: gotestsum --format short-verbose ./...
+
+      - name: Test
+        run: go test -v ./...
```

Comment:
I think this stage is already executed by the previous one
```suggestion

```

Reply:
Agree